Makefile

```diff
@@ -7,6 +7,8 @@ MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rules
 MAKEFLAGS += --no-print-directory
 BIN := .tmp/bin
+export PATH := $(BIN):$(PATH)
+export GOBIN := $(abspath $(BIN))
```

Comment:
In a subsequent PR, I'm planning to rip out our homegrown support for overriding the Go compiler in use. Now that 1.21 is out, users can set `GOTOOLCHAIN` without any special magic in this Makefile.